lectures/olg.md

```diff
@@ -463,12 +463,9 @@ def k_update(k, α, β):
 ```{code-cell} ipython3
 α, β = 0.5, 0.9
 kmin, kmax = 0, 0.1
-x = 1000
-k_grid = np.linspace(kmin, kmax, x)
-k_grid_next = np.empty_like(k_grid)
-
-for i in range(x):
-    k_grid_next[i] = k_update(k_grid[i], α, β)
+n = 1000
+k_grid = np.linspace(kmin, kmax, n)
+k_grid_next = k_update(k_grid,α,β)
```

Comment:
@longye-tian does `k_update` work on a vector? Just wondering why we are removing the loop here and not elsewhere (see below)